students/Dkuchan/ClassProject/PythonClassProject.py

```diff
@@ -0,0 +1,625 @@
+# PythonClassProject.py
+''' This python program is written for the advanced python programming class
+    it is a scouting program written for the First Robotics Competition based
+    on the 2017 game FIRST Steamworks. '''
+
+
+
+
+'''Need to add a tuple for allowed users to baseline who does what'''
+
+
+
+import math
+import os
+import csv
+import random
+
+matchlist = []
+teamlist = []   #HOLDS THE TEAM NUMBERS BECAUSE I COULD NOT GET A DIRECT LIST CREATION TO WORK
+teamlist2 = []  #HOLDS THE TEAM OBJECTS
+montelist = []
+teamdict = {}
+monteblue = [0, 0, 0]
+montered = [0, 0, 0]
+
+
+class Match:
+
+    # A match is a collection of data which represents a match FOR A SINGLE TEAM.
```

Comment:
not that I really understand the problem here, but it seems a "match" involved more than one team (2?) -- so you probably want a match object that captures that -- so you can't get things like "winner" out of sync.

Reply:
I would expect that the match object would have the teams involved in the match, and anything related to the match that isn't team-specific -- things like who won, the score, how long it took, or whatever.